Description: Fix the build on sid by including X11 libraries.
 Without -lX11, the build fails with the following symbols missing:
 XResetScreenSaver (in utils/fgviewer) and XConvertSelection (in src/Main)

Author: Markus Wanner <markus@bluegap.ch>
Forwarded: no
Last-Change: 2015-05-24

Index: flightgear-2017.2.1/src/Main/CMakeLists.txt
===================================================================
--- flightgear-2017.2.1.orig/src/Main/CMakeLists.txt	2017-05-22 15:45:21.235627168 +0200
+++ flightgear-2017.2.1/src/Main/CMakeLists.txt	2017-05-22 15:45:21.231627226 +0200
@@ -146,6 +146,7 @@
 	${SIMGEAR_LIBRARIES}
 	${OPENSCENEGRAPH_LIBRARIES}
 	${OPENGL_LIBRARIES}
+	${X11_LIBRARIES}
 	${PLIB_LIBRARIES}
 	${HLA_LIBRARIES}
 	${GDAL_LIBRARIES}
